maestro-ios/src/main/java/ios/LocalIOSDevice.kt

```diff
@@ -143,9 +143,8 @@ class LocalIOSDevice(
         return xcTestDevice.isScreenStatic()
     }
 
-    override fun setPermissions(id: String, permissions: Map<String, String>) {
-        simctlIOSDevice.setPermissions(id, permissions)
-        xcTestDevice.setPermissions(id, permissions)
+    override fun setPermissions(id: String, permissions: Map<String, String>): Result<Unit, Throwable> {
+        return simctlIOSDevice.setPermissions(id, permissions)
```

Comment:
This is breaking the notification change, and regresses maestro flow tester and the runs we do staging. I propose we take these rename changes in next release if these are not dependent on something important. Let me know your thoughts.

Reply:
Thanks for finding this! I have updated the cod to call both xctest and simctl in case of setPermissions